onnxruntime/contrib_ops/cpu/bert/embed_layer_norm.cc

```diff
@@ -124,13 +124,15 @@ Status EmbedLayerNorm<T>::Compute(OpKernelContext* context) const {
   }
 
   // Calculate mask
-  {
-    const int* mask_data = mask->template Data<int>();
+  if (nullptr != mask) {
```

Comment:
How many outputs are expected when no mask inputted. 1, or 2? if `mask` is null, `Tensor* mask_index = context->Output(1, mask_index_shape);` should not be called

Reply:
2.  Input mask is optional, but the output is not, which means we will always have mask_index output. When input mask is NULL, the mask_index output only has dummy result (all zero).